Promote "Retriable and non-retriable pod failures for Jobs" to Beta (#37242)
kubernetes/website: content/en/docs/concepts/workloads/pods/disruptions.md

## Pod disruption conditions {#pod-disruption-conditions}

{{< feature-state for_k8s_version="v1.26" state="beta" >}}

{{< note >}}
In order to use this behavior, you must have the `PodDisruptionConditions`
[feature gate](/docs/reference/command-line-tools-reference/feature-gates/)

> enabled in your cluster.

Reviewer: IMO, the previous version is more clear, but your change is also ok.

Author: ok, let's keep as is then.

`DeletionByPodGC`
: Pod, that is bound to a no longer existing Node, is due to be deleted by [Pod garbage collection](/docs/concepts/workloads/pods/pod-lifecycle/#pod-garbage-collection).

`TerminationByKubelet`

> : Pod is being terminated by Kubelet (either due to {{<glossary_tooltip term_id="node-pressure-eviction" text="Node Pressure Eviction">}} or [Graceful Node Shutdown](/docs/concepts/architecture/nodes/#graceful-node-shutdown)) and is due to be deleted by the controller owning the Pod.

Reviewer: ```suggestion
: Kubelet terminates the Pod, because of the {{<glossary_tooltip term_id="node-pressure-eviction" text="Node Pressure Eviction">}} or the [Graceful Node Shutdown](/docs/concepts/architecture/nodes/#graceful-node-shutdown). Pod is due to be deleted by the controller owning the Pod.
```

`DeletionByPodGC`
: Pod, that is bound to a no longer existing Node, is due to be deleted by [Pod garbage collection](/docs/concepts/workloads/pods/pod-lifecycle/#pod-garbage-collection).

`TerminationByKubelet`

> : Kubelet terminates the Pod, because of the {{<glossary_tooltip term_id="node-pressure-eviction" text="Node Pressure Eviction">}} or the [Graceful Node Shutdown](/docs/concepts/architecture/nodes/#graceful-node-shutdown). Pod is due to be deleted by the controller owning the Pod.

Reviewer 2: (nit)
This even is better written as:
```suggestion
: A kubelet requested termination of the Pod, because of either {{<glossary_tooltip term_id="node-pressure-eviction" text="node pressure eviction">}} or [graceful node shutdown](/docs/concepts/architecture/nodes/#graceful-node-shutdown).
```
